### ISSUE TITLE
problem with libcurl.dll

### ISSUE THREAD

**Issue by Naimer457** [adafcaefc/GDDLLLoader#2]
when i install the libcurl.dll, chrome cancels the download, is it safe?

**Comment by trxshonzaza** [comment]
> when i install the libcurl.dll, chrome cancels the download, is it safe?

hey uh librcurl.dll is now detected as ransomware so be careful i guess
![image](https://user-images.githubusercontent.com/89441166/130560339-3614ce70-6838-49c6-b3dd-d3494ba49e10.png)

**Comment by Pierrot-7** [comment]
> when i install the libcurl.dll, chrome cancels the download, is it safe?

I scanned the file on VirusTotal and it is detected as malicious by 41 out of 69 engines, so yeah, it's definitely not safe.

**Comment by mrxiaoman** [comment]
yes this modloader is not a virus remember it mods the reg fies without permission

**Comment by dom3** [comment]
libcurl.dll (I THINK) has always been in the game, if you make a fresh reinstall of Geometry Dash libcurl.dll should be there. It's detected as malicious because libcurl (https://curl.se/libcurl/) is a way to transfer URLs, so it wouldn't be a suprise as being detected for malicious intent
